.github/workflows/nightly-bnb.yml

```diff
@@ -0,0 +1,109 @@
+name: Self-hosted runner with slow tests with bnb from source (scheduled)
```

Comment:
Can we rename this and put BNB somewhere in the front. The issue is that https://github.com/huggingface/peft/actions cuts off long names, and from the visible snippet, it's not possible to see easily which one is the bnb nightly and which one is the PEFT nightly action.

Reply:
Agreed ! Done!